roles/st2smoketests/tasks/main.yml

```diff
@@ -49,3 +49,104 @@
   changed_when: no
   tags:
     - smoke-tests
+
+- name: Generate st2api key
+  command: st2 apikey create -k
+  environment:
+    ST2_AUTH_TOKEN: "{{ st2_token.stdout }}"
+  register: st2_api_key
+  changed_when: no
+  no_log: true
+  tags:
+    - smoke-tests
+
+- name: Set st2chatops_api_key
+  set_fact:
+    st2chatops_api_key: "{{ st2_api_key.stdout  }}"
+    st2chatops_hubot_adapter: "slack"
+    st2chatops_config: {"HUBOT_SLACK_TOKEN": "{{ hubot_token }}"}
+  changed_when: no
+  no_log: true
+  tags:
+    - smoke-tests
+
+- name: Install st2 pack from exchange
+  become: yes
+  environment:
+    ST2_AUTH_TOKEN: "{{ st2_token.stdout }}"
+  command: "st2 pack install st2"
+  changed_when: no
+  tags:
+    - smoke-tests
+
+- name: Update st2_api_key in st2chatops.env
+  become: yes
+  replace:
+    dest: /opt/stackstorm/chatops/st2chatops.env
+    regexp: '(?<=ST2_API_KEY=)(.*)$'
+    replace: '{{ st2chatops_api_key }}'
+  no_log: true
+  changed_when: no
+  notify: restart st2chatops
```

Comment:
This is a catch-22 situation. Either restart or idempotency. Removing this and https://github.com/StackStorm/ansible-st2/pull/125/files#diff-c6fa708e733eecbe8e8a283eb5690c19R124 in favor of: https://github.com/StackStorm/ansible-st2/pull/125/files#diff-c6fa708e733eecbe8e8a283eb5690c19R140